napari/_qt/layer_controls/qt_image_controls_base.py

```diff
@@ -4,7 +4,8 @@
 import numpy as np
 from qtpy.QtCore import Qt
 from qtpy.QtGui import QImage, QPixmap
-from qtpy.QtWidgets import QLabel, QPushButton, QSlider
+from qtpy.QtWidgets import QLabel, QPushButton
+from superqt import QLabeledDoubleSlider as QSlider
```

Comment:
Should we not use the "default" name of the widget from Qt, so that it may be more clear that it is not the stock `QSlider`?

Reply:
sure, can do.  was just too excited about the drop-in replacement :joy:

Reply:
It feels odd, is it a real exact drop-in replacement? Like all the methods are the same?

Reply:
that was the intention... and there are a fair number of tests to that extent... but, you know, I'm sure there's bugs somewhere :)